DTS-STN/Alpha-Site
# Changed all instances of experiments to projects.

# Description

- Changed all instances of `experiments` to `projects`.
- The old `experiments.js` page is now `projects/index.js`.
- The Backend Enpoint remains https://alphasite-api.dts-stn.com/experiments.
- The Backend Enpoint does not provide a "Coming soon" project so I commented out a test case under `__test__/pages/projects.test.js`.

## Acceptance Criteria

Nothing should of changed on the website.

## Test Instructions

1. Simply run the app and navigate through and see if anything crashes or a string is not displayed properly.
2. Remember that only the `experiment` keyword has been changed. 

## Checklist
